MDub329/ReboundLogParser
# Merge develop into main for 6.0 release

Testing:
* All buttons and combo boxes next to data grid should be disabled on load.
* CEF browser at bottom of window should load League Republic login page.
* Load log file(s) by both using the Open File dialog and drag-drop over data grid.
  * Load Players buttons should now enable if data grids have at least one data row.
* Find a results statistics page for a match in SPL League Republic site (must be logged in).
* Click load players button for each team.
  * Ensure combo boxes only activate if there are enough data rows in the team grid to support that box.
* Select players in combo boxes for each data row.
  * Test leaving at least one box empty to ensure Send button stays disabled.
  * Test all boxes filled per data row enables the send button for that team.
  * Selecting a duplicate player in more than one box will disable the send button and color the duplicated boxes red.
* Click the send button for each team when enabled.
  * Ensure the data table is correctly copied to the appropriate names selected in the CEF browser.
  * CRITICAL: Be sure to not click "UPDATE" button on League Republic page. Original data may be NON-RECOVERABLE!!!
    * Click the "[BackArrow] Results" link to safely back out of current data changed in the form.
* Click the "Clear Logs" button in the parser UI to ensure the data tables are cleared, all data labels (overtime win, not 3rd period, etc.) are cleared, and all but the load logs UI controls are disabled again as they were on initial form load.
